src/components/payment/Payment.test.js

```diff
@@ -71,11 +72,14 @@ const defaultAutoVoucherQuery = {
 }
 
 const mountPaymentSection = ({
+  trainingData = defaultTrainingData,
   paymentMutation,
   autoVoucherQuery = defaultAutoVoucherQuery,
   validateVoucherQuery,
   validateVatQuery,
+  meetup = false,
```

Comment:
@ric9176 what if tomorrow we decide to show the newsletter checkbox also in the case of the 1-day workshops? Maybe we should say if we want the checkbox displayed and not the reason since the later can change. What about a prop called showSubscribeToNewsletter ?